FlashDevelop/Docking/TabbedDocument.cs

```diff
@@ -468,6 +468,13 @@ public void Reload(Boolean showQuestion)
                 this.SciControl.IsReadOnly = FileHelper.FileIsReadOnly(this.FileName);
                 this.SciControl.SetSel(position, position);
                 this.SciControl.EmptyUndoBuffer();
+
+                List<Int32> bookmarksCopy = this.bookmarks;
```

Comment:
You're right. I'm still fairly inexperienced with C# since I mainly use C++ so I thought that worked. I stepped through it again and it turns out that line isn't event needed since OnMarkerChanged() prevents it from adding duplicates to this.bookmarks, which I thought was an issue.